src/fracas/syntax.ts

```diff
@@ -181,6 +181,37 @@ export function completionKind(fracasKind: FracasDefinitionKind): vscode.Complet
     }
 }
 
+export function symbolKind(fracasKind: FracasDefinitionKind): vscode.SymbolKind {
+    switch (fracasKind) {
+        case (FracasDefinitionKind.enum):
+            return vscode.SymbolKind.Enum;
+        case (FracasDefinitionKind.gameData):
+            return vscode.SymbolKind.Module;
+        case (FracasDefinitionKind.key):
+            return vscode.SymbolKind.Variable;
+        case (FracasDefinitionKind.mask):
+            return vscode.SymbolKind.Enum;
+        case (FracasDefinitionKind.typeOptional):
+            return vscode.SymbolKind.Struct;
+        case (FracasDefinitionKind.type):
+            return vscode.SymbolKind.Struct;
+        case (FracasDefinitionKind.variant):
+            return vscode.SymbolKind.Struct;
+        case (FracasDefinitionKind.variantOption):
+            return vscode.SymbolKind.Struct;
+        case (FracasDefinitionKind.syntax):
+            return vscode.SymbolKind.Function;
+        case (FracasDefinitionKind.define):
+            return vscode.SymbolKind.Variable;
+        case (FracasDefinitionKind.field):
+            return vscode.SymbolKind.Variable;
+        case (FracasDefinitionKind.unknown):
```

Comment:
this is missing the new `FracasDefinitionKind.Text` from this PR:
https://github.com/WSStudios/vscode-fracas/pull/1/files